Cargo.toml

```diff
@@ -31,7 +31,7 @@ async-std = { version = "1.9.0", features = ["attributes"] }
 async-trait = "0.1.50"
 clap = "2.33.3"
 combinations = "0.1.0"
-flux = { git = "https://github.com/influxdata/flux", tag = "v0.125.0" }
+flux = { git = "https://github.com/influxdata/flux", tag = "v0.127.1" }
```

Comment:
We probably want to point this at whatever version of flux has your changes in it, otherwise I think we will get the same conflicts we saw yesterday.